app/src/test/java/com/concordia/mcga/adapters/DirectionArrayAdapterTest.java

```diff
@@ -0,0 +1,51 @@
+package com.concordia.mcga.adapters;
+
+import android.content.Context;
+import android.view.View;
+
+import com.concordia.mcga.activities.BuildConfig;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.robolectric.RobolectricGradleTestRunner;
+import org.robolectric.annotation.Config;
+
+import java.util.ArrayList;
+
+import static junit.framework.Assert.assertNotNull;
+import static junit.framework.Assert.assertNull;
+import static org.mockito.Mockito.when;
+
+/**
+ * Created by Charmander on 3/12/2017.
+ */
+@RunWith(RobolectricGradleTestRunner.class)
+@Config(constants = BuildConfig.class,sdk = 21)
+
+public class DirectionArrayAdapterTest {
+    // Array Adapter
+    private DirectionsArrayAdapter adapter = null;
+    Context context;
+    private ArrayList<String> directionsText = new ArrayList<String>();
+    private ArrayList<String> directionsImage = new ArrayList<String>();
+
+    @Before
+    public void setUp() throws Exception {
+        context = Mockito.mock(Context.class);
+        when(context.getSystemService(Context.LAYOUT_INFLATER_SERVICE)).thenReturn(null);
+        adapter = new DirectionsArrayAdapter(context, directionsText, directionsImage);
+    }
+
+    @Test
+    public void Adapter_IsNull_True(){
+        assertNotNull(adapter);
+    }
+
+    @Test
+    public void GetView_IsNotNull_True(){
```

Comment:
The reason is that I don't know how to test this method... I don't know which parameter to pass to it. Maybe Mark knows but he didn't answer me